index.js

```diff
@@ -3,7 +3,7 @@
 var BufferStreams = require('bufferstreams');
 var htmlmin = require('html-minifier');
 var gutil = require('gulp-util');
-var objectAssign = require('object-assign');
```

Comment:
Anyway, we can remove any Object.assign-like modules from the dependencies if we drop Node 0.x support of this plugin.

Is there any plans to drop Node v0.x support? For reference https://github.com/nodejs/LTS/#lts-plan, 

> The existing joyent/node v0.12 will continue in LTS for a period of 6 months after the initial LTS release from the converged repo

I think Apr. 1, 2016 is the time to start using `Object.assign` directly, remove polyfill and publish a new major version.